src/utils.rs

```diff
@@ -80,6 +83,67 @@ pub fn gen_items<const N: usize>(seed: Vec<u8>, set_size: usize) -> Vec<[u8; N]>
     s_p
 }
 
+/// Generate a set of weighted items given the total weight size, number of items and a seed
```

Comment:
What is an item in this context? I'd like to better understand the purpose of this function and how it is used. You seem to be hashing the seed + the current index.

Reply:
This function is under the "// test & Bench helpers" comment, is that not enough?

I'll create a test folder sure.

p.s. The weighted function will be removed, that was an oversight.